Deck_Shuffler/Deck_Shuffler/Deck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Deck
+    {
+        public List<Card> Cards = new List<Card>();
+        
+        public Deck()
+        {
+            string[] suits = new string[] { "Hearts", "Diamonds", "Clubs", "Spades" };
+            string[] values = new string[] { "Ace", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten", "Jack", "Queen", "King" };
```

Comment:
Also why are you using arrays. Use Lists wherever you can! Using arrays will make you look like a noob!